PowerEditor/src/Parameters.h

```diff
@@ -1939,6 +1942,11 @@ class NppParameters final
 
 	LanguageNameInfo getLangNameInfoFromNameID(const std::wstring& langNameID);
 
+	void setNbTabCompactLabelLen(UINT nb) {
+		_nppGUI._tabCompactLabelLen = nb;
+	};
+	UINT getNbTabCompactLabelLen() const { return _nppGUI._tabCompactLabelLen; };
+

```

Comment:
@xomx 
Since now the rule is changed to conform to the  "-Wextra-semi" flag, it would be good to remove the superfluous `;` from the 2 function definitions above. 
@ozone10 could you add the flag on our CI GCC build & correct the codebase?